content_scripts/ui_component.coffee

```diff
@@ -72,6 +72,8 @@ class UIComponent
                 when "setIframeFrameId" then @iframeFrameId = event.data.iframeFrameId
                 when "hide" then @hide()
                 else @handleMessage event
+      if Utils.isFirefox()
```

Comment:
Add a newline before this line, to separate it from the very long block before it.